website/src/components/TeamProfileCards/index.tsx

```diff
@@ -139,6 +120,33 @@ export function ActiveTeamRow(): JSX.Element {
 export function HonoraryAlumniTeamRow(): JSX.Element {
   return (
     <div className="row">
+      <TeamProfileCardCol
```

Comment:
Maybe don't make this change in this PR because I'm not sure how to refactor it and present things correctly.

But it doesn't feel right to present Joel (creator of Docusaurus v1, former director of Meta OSS and now Rust foundation) as a former student 😅 

I'll handle this myself later

Reply:
@slorber "Alumni" refers to any former team member; it's not necessarily a student.

Reply:
ah ok didn't know 🤪

so we can keep it this way then 👍